In [1]:
import cv2

In [2]:
image = cv2.imread('What-is-data-science-2.jpg')

In [3]:
image.shape

(503, 1000, 3)

here it shows that 503 is height of the image, 1000 is width of the image and 3 are color channels of the dimensions